client/src/plugins/user-journey-statistics/event-handlers/LinkEventHandler.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+const TRACKED_LINK_PARENT_IDS = [ 'version-info-overlay', 'welcome-page-cloud', 'welcome-page-platform', 'welcome-page-learn-more' ];
+
+export default class LinkEventHandler {
+  constructor(props) {
+    this.track = props.track;
+
+    document.addEventListener('click', this.trackClick);
+  }
+
+   trackClick = ({ target }) => {
```

Comment:
This is a 90% copy of what is done for ET. I duplicated it since they are two different (built-in) plugins and we might differ strategy for each tracking platform. Also because it didn't seem right to include a specific utility function for this in the core (so it could be shared and accessed by both). Let me know what you think.

Reply:
> Also because it didn't seem right to include a specific utility function for this in the core (so it could be shared and accessed by both).

If we don't know yet whether we want to track the same events in ET and Mixpanel, it's fine to have it duplicated and independent for now. What I'm wondering is, can't we have a generic utility to extract the payload for a given list of IDs?

```js
const { payloadType, payload } = retrieveLinkPayload(target, TRACKED_LINK_PARENT_IDS);

if(payloadType) { ... }
```